### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by c4rlo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -515,36 +515,36 @@ Optional parameters to provide more metadata:
 - `-N Name`
 - `-C "Compositor description"`
 
-Arguments and metadata are stored in specifier unit drop-ins if needed.
```
Comment (c4rlo) [inline README.md:518]: Here I wasn't sure what to change this to; it seems to work fine without this word.

File: README.md [hunk 2]
```diff
@@ -362,21 +362,21 @@ uwsm forks a process that probes systemd activation environment for
...
 A separate unit, `wayland-session-waitenv.service` is launched alongside
 compositor, with similar ordering after `graphical-session-pre.target`, before
 `graphical-session.target`. It also waits for the same variables in the same
-manner, then successfully exits (or times out). Its job is to dealy
+manner, then successfully exits (or times out). Its job is to delay
 `graphical-session.target` activation in case compositor signals its readiness
 prematurely. Or to fail startup if expected vars do not appear.
 
-`uwsm finalize` command fills systemd and dbus environments with essential vars
+The `uwsm finalize` command fills systemd and D-Bus environments with essential vars
```
Comment (Vladimir-csp) [inline README.md:376]: I try to observe 80 char line limit in md and scd where possible, could you please reflow long lines?

File: README.md (hunk 1 above)
Comment (Vladimir-csp) [reply README.md:518]: Specifier is a term from systemd documentation.

**Comment by Vladimir-csp** [comment]
I do make a lot of typos :(
Thanks!

**Pull request merged**